crates/core/src/config.rs

```diff
@@ -36,15 +38,23 @@ pub const SEPOLIA_CONSENSUS_RPC: &str =
     "http://unstable.sepolia.beacon-api.nimbus.team";
 pub const SEPOLIA_FALLBACK_RPC: &str = "https://sync-sepolia.beaconcha.in";
 
+enum Blockchain {
+    Ethereum,
+    Starknet,
+}
+
 /// global config
-#[derive(Clone, Deserialize, Debug)]
+#[derive(Clone, Deserialize, Debug, Validate)]
 pub struct Config {
     pub network: Network,
+    #[validate(url)]
     pub eth_execution_rpc: String,
+    #[validate(url)]
     pub starknet_rpc: String,
     #[serde(default = "data_dir")]
     pub data_dir: PathBuf,
     #[serde(default = "poll_secs")]
+    #[validate(range(min = 1, max = 10))]
```

Comment:
I have fixed it to a range of 10 as a starting point for discussion. I am also not sure which value it should be, please add suggestion

Reply:
The goal is to avoid unreasonably high poll rate that would still be a valid `u64` value. Maybe 1h (3600 seconds) is such a reasonable bound. We can always update it if necessary.

Reply:
Ok, will set min = 1, max = 3600